docker/cloudsim_sim/Dockerfile

```diff
@@ -108,7 +107,7 @@ RUN rosdep update
 # docker is run
 RUN mkdir -p subt_ws/src \
  && cd subt_ws/src \
- && hg clone https://bitbucket.org/osrf/subt
+ && git clone git@github.com:osrf/subt
```

Comment:
use https? this requires ssh keys to be set up on github

Reply:
same goes for `subt_shell` and `subt_sim_entry`

Reply:
Changed in  0a4d7d0be79a550403bd6718a14889773090455e